### PR TITLE
Some linting and ros-lib update

### PR DESCRIPTION
**Pull request by strongly-typed**

### CODE REVIEW

File: examples/generic/ros/can_bridge/project.xml [hunk 1]
```diff
@@ -2,6 +2,7 @@
   <extends>modm:blue-pill</extends>
   <options>
     <option name="modm:build:build.path">../../../../build/generic/ros/can_bridge</option>
+    <option name="modm:build:openocd.cfg">openocd.cfg</option>
```
Comment (salkinium) [inline examples/generic/ros/can_bridge/project.xml:5]: You still need to actually add an `openocd.cfg` file. See https://github.com/modm-io/modm/pull/240/files for the recommended content.

Comment (strongly-typed) [reply examples/generic/ros/can_bridge/project.xml:5]: Just copied from the blue pill example 😂

**Comment by salkinium** [comment]
Better commit message
> [printf] Cast float argument to double explicitly

**Review by salkinium — APPROVED** [review]
Excellent!

**Pull request merged**